### ISSUE TITLE
[BUG] Imviz: NotImplementedError: ROIs of type EllipticalROI are not yet supported

### ISSUE THREAD

**Issue by pllim** [spacetelescope/jdaviz#569]
**Describe the bug**
Running `ImvizExample.ipynb` gives `NotImplementedError: ROIs of type EllipticalROI are not yet supported`.

**To Reproduce**
1. Run `notebooks/ImvizExample.ipynb`.
2. Run first 3 cells to get the viewer to show up with loaded data.
3. Check the checkbox to get one of the data to show (doesn't matter which one).
4. Click on the circle selection and select something. A pinkish circle should show up in the display.
5. Jump to this cell and run it: `regions = imviz.app.get_subsets_from_viewer('viewer-1')`

```
.../jdaviz/jdaviz/app.py in get_subsets_from_viewer(self, viewer_reference, data_label)
    488                 #  therefore, if the data is already 2d, simply use as is.
    489                 if value.data.ndim == 2:
--> 490                     region = value.data.get_selection_definition(
    491                         format='astropy-regions')
    492                     regions[key] = region

.../glue/core/data.py in get_selection_definition(self, subset_id, format, **kwargs)
    371         handler = subset_state_translator.get_handler_for(format)
    372 
--> 373         return handler.to_object(subset, **kwargs)
    374 
    375 

.../glue_astronomy/translators/regions.py in to_object(self, subset)
     98 
     99             else:
--> 100                 raise NotImplementedError("ROIs of type {0} are not yet supported"
    101                                           .format(roi.__class__.__name__))
    102 

NotImplementedError: ROIs of type EllipticalROI are not yet supported
```

**Expected behavior**
No error and `regions` show me the region object(s).

**Screenshots**
N/A

**Desktop (please complete the following information):**
 - OS: Red Hat Enterprise Linux Server release 7.9 (Maipo)
 - Browser: Chrome 90.0.4430.85 (Official Build) (64-bit)
 - Jupyter: [run "jupyter --version"]

```
jupyter core     : 4.6.3
jupyter-notebook : 6.0.3
qtconsole        : 5.0.2
ipython          : 7.15.0
ipykernel        : 5.3.2
jupyter client   : 6.1.6
jupyter lab      : 2.1.5
nbconvert        : 6.0.7
ipywidgets       : 7.6.3
nbformat         : 5.0.7
traitlets        : 5.0.5
```

**Package versions (please complete the following information):**

Linux-3.10.0-1160.21.1.el7.x86_64-x86_64-with-glibc2.10
Python 3.8.3 (default, May 19 2020, 18:47:26) 
[GCC 7.3.0]
Numpy 1.19.2
astropy 4.3.dev1193+g9d8014c82
specutils 1.3.dev42+g57dd2b28
spectral-cube 0.5.0
pyyaml 5.4.1
click 7.1.2
asteval 0.9.23
idna 2.10
traitlets 5.0.5
bqplot 0.12.21
bqplot-image-gl 1.3.0
glue-core 1.0.1
glue-jupyter 0.3
glue-astronomy 0.1 (tried both stable and dev, they both give the same version number)
echo 0.5
ipyvue 1.5.0
ipyvuetify 1.6.1
ipysplitpanes 0.1.0
ipygoldenlayout 0.3.0
voila 0.2.6
vispy 0.6.6
Jdaviz 1.0.4.dev472+ga48cf08.d20210426

**Comment by pllim** [comment]
p.s. Rectangle selection works fine.

```python
>>> regions
{'Subset 1': <RectanglePixelRegion(center=PixCoord(x=279.83572635135135, y=383.5854247104247),
width=116.91891891891888, height=52.81853281853273, angle=0.0 deg)>}
```

**Comment by astrofrog** [comment]
The issue is due to the following logic:

https://github.com/glue-viz/glue-jupyter/blob/5149719e96f4e8d1d9a88fe84b3834154362fdd9/glue_jupyter/bqplot/common/tools.py#L185-L188

For some reason the shape drawn is currently slightly elliptical even though it should be circular. We could also just add support for elliptical regions in glue-astronomy but the point here is that we *should* get back a circular ROI.

I'm currently investigating what could be going on.

**Comment by astrofrog** [comment]
@pllim - I tracked this down, can you try with https://github.com/glue-viz/glue-jupyter/pull/217 to see if that fixes it for you?

**Comment by astrofrog** [comment]
Huh I did not

**Comment by pllim** [comment]
> I did not know issues could be closed across repos!

You knew back in 2014... 😆 https://github.com/astropy/package-template/issues/81#issuecomment-45835589

**Comment by pllim** [comment]
Confirmed fixed with `glue-jupyter 0.4.dev4+gbd0cb1c`. It will need a release and then we need to bump minversion here.

```python
{'Subset 1': <CirclePixelRegion(center=PixCoord(x=108.63302612304688, y=503.1221008300781), radius=72.01611328125)>}
```
